### ISSUE TITLE
Test error in unittest_windows

### ISSUE THREAD

**Issue by jimchen90** [pytorch/audio#741]
## 🐛 Bug

There are test errors (as shown below) in PR #738. I also saw it in my PR #735 .

## To Reproduce
This is the [error message.](https://app.circleci.com/pipelines/github/pytorch/audio/2421/workflows/670c901e-48cf-4ea4-8ae6-38c3ee8dd22a/jobs/63894/steps)
```
#!bash.exe
.circleci/unittest/windows/scripts/run_test.sh
Collecting environment information...
PyTorch version: 1.6.0.dev20200623
Is debug build: No
CUDA used to build PyTorch: None

OS: Microsoft Windows Server 2019 Datacenter
GCC version: Could not collect
CMake version: Could not collect

Python version: 3.6
Is CUDA available: No
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA

Versions of relevant libraries:
[pip3] numpy==1.18.5
[pip3] torch==1.6.0.dev20200623
[pip3] torchaudio==0.6.0a0+bbf3003
[conda] Could not collect
============================= test session starts =============================
platform win32 -- Python 3.6.10, pytest-5.4.3, py-1.8.2, pluggy-0.13.1 -- C:\Users\circleci\project\env\python.exe
cachedir: .pytest_cache
rootdir: C:\Users\circleci\project
plugins: cov-2.10.0
collecting ... collected 1790 items / 1 error / 1789 selected

=================================== ERRORS ====================================
_____________ ERROR collecting test/test_librosa_compatibility.py _____________
test\test_librosa_compatibility.py:15: in <module>
    import librosa
env\lib\site-packages\librosa\__init__.py:12: in <module>
    from . import core
env\lib\site-packages\librosa\core\__init__.py:125: in <module>
    from .time_frequency import *  # pylint: disable=wildcard-import
env\lib\site-packages\librosa\core\time_frequency.py:11: in <module>
    from ..util.exceptions import ParameterError
env\lib\site-packages\librosa\util\__init__.py:77: in <module>
    from .utils import *  # pylint: disable=wildcard-import
env\lib\site-packages\librosa\util\utils.py:5: in <module>
    import scipy.ndimage
env\lib\site-packages\scipy\__init__.py:104: in <module>
    from . import _distributor_init
env\lib\site-packages\scipy\_distributor_init.py:61: in <module>
    WinDLL(os.path.abspath(filename))
env\lib\ctypes\__init__.py:348: in __init__
    self._handle = _dlopen(self._name, mode)
E   OSError: [WinError 126] The specified module could not be found
============================== warnings summary ===============================
env\lib\site-packages\_pytest\junitxml.py:417
  C:\Users\circleci\project\env\lib\site-packages\_pytest\junitxml.py:417: PytestDeprecationWarning: The 'junit_family' default value will change to 'xunit2' in pytest 6.0.
  Add 'junit_family=xunit1' to your pytest.ini file to keep the current format in future versions of pytest and silence this warning.
    _issue_warning_captured(deprecated.JUNIT_XML_DEFAULT_FAMILY, config.hook, 2)

torchaudio\extension\extension.py:14
  C:\Users\circleci\project\torchaudio\extension\extension.py:14: UserWarning: torchaudio C++ extension is not available.
    warnings.warn('torchaudio C++ extension is not available.')

-- Docs: https://docs.pytest.org/en/latest/warnings.html
---- generated xml file: C:\Users\circleci\project\test-results\junit.xml -----

---------- coverage: platform win32, python 3.6.10-final-0 -----------
Name                                      Stmts   Miss  Cover
-------------------------------------------------------------
torchaudio\__init__.py                       14      4    71%
torchaudio\_internal\__init__.py              0      0   100%
torchaudio\_internal\misc_ops.py             17     12    29%
torchaudio\_internal\module_utils.py         25      4    84%
torchaudio\backend\__init__.py                5      0   100%
torchaudio\backend\common.py                 32     11    66%
torchaudio\backend\no_backend.py             17      4    76%
torchaudio\backend\soundfile_backend.py      61     29    52%
torchaudio\backend\sox_backend.py            98     67    32%
torchaudio\backend\sox_io_backend.py          5      1    80%
torchaudio\backend\utils.py                  41     15    63%
torchaudio\compliance\__init__.py             1      0   100%
torchaudio\compliance\kaldi.py              307    269    12%
torchaudio\datasets\__init__.py              10      0   100%
torchaudio\datasets\cmuarctic.py             48     30    38%
torchaudio\datasets\commonvoice.py           50     31    38%
torchaudio\datasets\gtzan.py                 54     35    35%
torchaudio\datasets\librispeech.py           50     34    32%
torchaudio\datasets\ljspeech.py              41     24    41%
torchaudio\datasets\speechcommands.py        44     28    36%
torchaudio\datasets\utils.py                176    138    22%
torchaudio\datasets\vctk.py                  63     43    32%
torchaudio\datasets\yesno.py                 46     30    35%
torchaudio\extension\__init__.py              3      0   100%
torchaudio\extension\extension.py            30     10    67%
torchaudio\functional.py                    793    739     7%
torchaudio\kaldi_io.py                       28      9    68%
torchaudio\models\__init__.py                 2      0   100%
torchaudio\models\_wavernn.py               101     81    20%
torchaudio\models\wav2letter.py              18     11    39%
torchaudio\sox_effects\__init__.py            4      1    75%
torchaudio\sox_effects\sox_effects.py        87     62    29%
torchaudio\transforms.py                    331    250    24%
torchaudio\version.py                         2      0   100%
-------------------------------------------------------------
TOTAL                                      2604   1972    24%

=========================== short test summary info ===========================
ERROR test/test_librosa_compatibility.py - OSError: [WinError 126] The specif...
!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================== 2 warnings, 1 error in 2.96s =========================
```

## Expected behavior
Test passed.

**Comment by peterjc123** [comment]
The workground is pin scipy to 1.4.1.
Related: https://github.com/scipy/scipy/issues/12400 https://github.com/pytorch/pytorch/issues/40366

**Comment by vincentqb** [comment]
Relates to #742

**Comment by vincentqb** [comment]
Closed by #743
